tests/functional/curate/cram/metadata-input.t

```diff
@@ -57,6 +57,21 @@ Test CSV metadata input from stdin
   {"strain": "sequence_B", "country": "USA", "date": "2020-10-02"}
   {"strain": "sequence_C", "country": "USA", "date": "2020-10-03"}
 
+Test Excel (.xls) metadata input
```

Comment:
worth adding an OpenOffice doc test too?

Reply:
Sure. (I opted not to bother and wondered if someone would ask for it. :upside_down_face:)

Reply:
[Done](https://github.com/nextstrain/augur/compare/49414435d641e72ce8454f591bf885d8e796433e..cdae769bcff674675eaf72d835206b474aadba67).